Documentation/sdn-features-packet-transforms.md

```diff
@@ -217,6 +217,8 @@ Etc…
 
 
  ## Packet Flow
+For the first packet of a TCP flow, we take the Slow Path, running the transposition engine and matching at each layer.  For subsequent packets, we take the Fast Path, 
+matching a unified flow via UFID and applying a transposition directly against rules.
```

Comment:
What is a UFID? If it's a MSFT-specific buzzword, might want to reword it to be generally meaningful. Should be in glossary.

Reply:
UFID is a Unified Flow ID.  Can put into the Glossary.